### ISSUE TITLE
move all tests to aiida 0.11.2

### ISSUE THREAD

**Issue by ltalirz** [aiidateam/aiida_docker_compose#13]
- [ ] cp2k
- [ ] siesta
- [ ] yambo

**Comment by ltalirz** [comment]
@giovannipizzi no idea how much work this is, perhaps it would make sense to get the help of other developers here - let me know

**Comment by giovannipizzi** [comment]
Yes, should be easy but maybe better to ask the various plugin developers, also because maybe this repo contains very outdated versions of the plugins
